_data/nav.yml

```diff
@@ -479,6 +479,7 @@
     - url: "#API"
     - url: "#Events"
     - url: "#Commands"
+    - url: "#Queryable values"
```

Comment:
"Query commands" or "Query value commands" might make more sense? "Queryable values" sounds rather wrong/incorrect IMO and we should have "command" in there somewhere.

Reply:
I see later you used "Queryable command values" which would be better, but I still find "Queryable" odd 🤷

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1590/commits/c4f6e844ae7168e5343fd3e281d776e5355b09ac